botlistbot/components/basic.py

```diff
@@ -246,7 +246,7 @@ def register(dp):
     dp.add_error_handler(error)
     dp.add_handler(
         MessageHandler(
-            Filters.text & Filters.group,
+            Filters.text,
```

Comment:
Hmm, but why 🤔

Reply:
The handler has a check for channel posts which calls another function. That other function adds the channel to the database if the channel isn't already in there.

Reply:
The channel post check adds the channel to the database, which is helpful for when you run it for the first time.

Reply:
Oh, you removed it. Gotta say, didn't know I had red/green weakness 😅